### DISCUSSION TITLE
Unable to download backup data files from SERNEC and Bryophyte Portal

### DISCUSSION THREAD

**Discussion by mgoliver22** [Symbiota/community-board#521] [Problems/Bugs]
I've been trying to download backup data files from SERNEC and the Bryophyte Portal over the past few business days without any luck. A new window will pop up, like normal, and I select to Perform Backup. After a minute though this window will just turn white and even if I leave it up for a long time a backup file never downloads. I am able to successfully download a backup data file from the Lichen Portal.

![Screenshot 2024-05-14 162453](https://github.com/BioKIC/symbiota-docs/assets/66650858/6f052c7e-e3b6-4522-8c4a-53a34b8af81e)
![Screenshot 2024-05-14 162513](https://github.com/BioKIC/symbiota-docs/assets/66650858/8c4d0543-fdfc-4b1a-856c-2b65827babba)

**Comment by themerekat** [comment]
Hi @mgoliver22 , can you try downloading the files as UTF-8?

**Reply by mgoliver22** [reply]
Yeah, the same thing happens.

**Reply by themerekat** [reply]
Ok thanks for checking!

**Comment by themerekat** [comment]
@mgoliver22 , I believe this bug is now fixed. Give it a whirl!

**Reply by mgoliver22** [reply]
I'm still having the same problem. I tried it using Google Chrome, just in case, and it still won't download but instead of the download window turning white it gives an error. I've tried downloading as ISO-8859-1 and UTF-8.
![Screenshot 2024-05-16 152717](https://github.com/BioKIC/symbiota-docs/assets/66650858/92315b5e-0263-45bf-adba-f3b642e7fb33)

**Comment by themerekat** [comment]
@egbot, for your investigation!
